test/utils/jwtUtil.test.js

```diff
@@ -0,0 +1,30 @@
+const chai = require('chai');
+const jwtUtil = require('../../lib/utils/jwtUtil');
+
+const { expect } = chai;
+const expirationTime = '1h',
+  userId = 'd4d6884c-ed49-4158-99cd-6305af05a4a6';
+let token, verifiedToken;
+
+describe('Test for creation and verification of token', () => {
+  it('Should have methods to generate and verify token', () => {
+    expect(jwtUtil.generateToken).to.be.a('function');
+    expect(jwtUtil.verifyToken).to.be.a('function');
+  });
+
+  it('should return generated token and decode the token', () => {
+    before(() => {
```

Comment:
I would love to know why you are making assertions in a `before` function.

Reply:
I just read about it. The info on mocha docs said it runs that block before anything. I did not realise it's not for assertions. correction taken